rekolekcje-engine/src/main/resources/db/changelog/001-initial.xml

```diff
@@ -1,5 +1,7 @@
 <?xml version="1.1" encoding="UTF-8" standalone="no"?>
```

Comment:
to nie jest dobry pomysł, żeby tu poprawiać formatowanie. Wtedy taki commit zepsuje wszystkim bazę, po to mamy liquibase, żeby ją bezpiecznie ewoluować, a nie za każdym razem robić drop all. Dlatego pomijając commity, które celowo porządkują changestety liquibase powinniśmy zdecydowanie unikać modyfikacji już istniejących (nawet pojedynczy znak zmieni sumę kontrolną).